.travis.yml

```diff
@@ -91,11 +91,20 @@ before_install:
     - pwd
     - mkdir /tmp/cozy-desktop
     - ln -s /tmp/cozy-desktop tmp
+    - npm install -g yarn
```

Comment:
Another question: any concern on your side with the `npm install -g` instead of using the ubuntu repos? I was not very comfortable about mixing deb packaged node from third-party repo with nvm... But I'm not comfortable about using a non-recommended installation method either.